app/src/main/java/us/kikin/apps/android/bgplayer/ui/show/ShowHeaderViewHolder.kt

```diff
@@ -1,36 +1,32 @@
 package us.kikin.apps.android.bgplayer.ui.show
 
 import android.view.LayoutInflater
-import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageView
-import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import coil.load
-import com.google.android.material.button.MaterialButton
-import us.kikin.apps.android.bgplayer.R
+import us.kikin.apps.android.bgplayer.databinding.ItemShowHeaderBinding
 import us.kikin.apps.android.bgplayer.models.VideoShowModel
 
-class ShowHeaderViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+class ShowHeaderViewHolder(
+    private val binding: ItemShowHeaderBinding
+) : RecyclerView.ViewHolder(binding.root) {
     companion object {
         fun from(parent: ViewGroup): ShowHeaderViewHolder {
-            val layoutInflater = LayoutInflater.from(parent.context)
-            val view = layoutInflater.inflate(R.layout.item_show_header, parent, false)
-            return ShowHeaderViewHolder(view)
+            val inflater = LayoutInflater.from(parent.context)
+            val binding = ItemShowHeaderBinding.inflate(inflater, parent, false)
+            return ShowHeaderViewHolder(binding)
         }
     }
 
-    private val showThumbnail: ImageView = view.findViewById(R.id.show_thumbnail)
-    private val showName: TextView = view.findViewById(R.id.show_name)
-    private val followButton: MaterialButton = view.findViewById(R.id.show_follow_button)
-
     fun bind(item: VideoShowModel, showClickListener: ShowItemClickListener) {
-        showName.text = item.name
-        showThumbnail.load(item.imageUrl) {
-            crossfade(true)
-        }
-        followButton.setOnClickListener {
-            showClickListener.onFollowShowClicked(item.id)
+        with(binding) {
+            showName.text = item.name
+            showThumbnail.load(item.imageUrl) {
+                crossfade(true)
+            }
+            showFollowButton.setOnClickListener {
+                showClickListener.onFollowShowClicked(item.id)
+            }
```

Comment:
These would be cleaner if the lambda was on the same line as the method call

Reply:
I would generally agree. But this is how it's written in the official docs.. Maybe there's a reason?
https://developer.android.com/topic/libraries/view-binding#fragments